pkg/assembler/backends/arangodb/vulnerability.go

```diff
@@ -0,0 +1,318 @@
+//
+// Copyright 2023 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package arangodb
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/arangodb/go-driver"
+	"github.com/guacsec/guac/pkg/assembler/graphql/model"
+)
+
+type dbVulnID struct {
+	TypeID   string `json:"type_id"`
+	VulnType string `json:"type"`
+	VulnID   string `json:"vuln_id"`
+	Vuln     string `json:"vuln"`
+}
+
+type dbVulnType struct {
+	TypeID   string `json:"type_id"`
+	VulnType string `json:"type"`
+}
+
+type VunIds struct {
```

Comment:
nit : typo Vuln?

Reply:
fixed and made lower case as no reason to expose.